agent/consul/acl_endpoint.go

```diff
@@ -65,26 +67,29 @@ func (a *ACL) fileBootstrapResetIndex() uint64 {
 	raw, err := ioutil.ReadFile(path)
 	if err != nil {
 		if !os.IsNotExist(err) {
-			a.srv.logger.Printf("[ERR] acl.bootstrap: failed to read %q: %v", path, err)
+			a.logger.Error("failed to read path",
```

Comment:
You've lost any indication this is related to acl bootstrapping from this error message. Probably either do a sub-logger like the original or just adjust the string message a bit?

Reply:
It does output `agent.acl` - but I can add  "bootstrap" to the front of it - is that what you're requesting?

Reply:
* will re-add bootstrap